content/python/concepts/sets/terms/issuperset/issuperset.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: '.issuperset()'
+Description: 'Checks whether all elements in specified set exist in the original set.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Collections'
+  - 'Methods'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.issuperset()`** method returns `True` if all the items in the specified set exists in the original set,otherwise it returns `False`.
+
+## Syntax
+
+```pseudo
+set1.issuperset(set2)
+```
+
+## Another Syntax
+
+```pseudo
+set1 >= set2
+```
```

Comment:
This can be added in a note format, but make sure that you add it below the parameter description, use the following:

>**Note:** In Python, the `>=` operator can be used to check if one set is a superset of another, functioning equivalently to the `.issuperset()` method.